addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/settings/YamahaSettingsUtil.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.settings;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.function.Consumer;
+
+/**
+ * Utility to work with {@link Thing} settings.
+ *
+ * @author Tomasz Maruszak - Initial contribution
+ */
+public class YamahaSettingsUtil {
+
```

Comment:
Hm, this class should go in my opinion. Configuration should be read/parsed/assigned to fields by the Configuration ESH class. If required configuration is missing, the mentioned class will complain.